tests/acceptance/AuthorshipCest.php

```diff
@@ -0,0 +1,153 @@
+<?php
+/**
+ * Tests for core module's authorship features.
+ *
+ * phpcs:disable WordPress.Files, WordPress.NamingConventions, PSR1.Classes.ClassDeclaration.MissingNamespace, HM.Functions.NamespacedFunctions
+ */
+
+use Codeception\Util\Locator;
+
+/**
+ * Test core module admin features.
+ */
+class AuthorshipCest {
+
+	/**
+	 * Rollback callback for the authorship activation bootstrap call.
+	 *
+	 * @var callable
+	 */
+	protected $rollback = null;
+
+	/**
+	 * Make sure Authorship is activated.
+	 *
+	 * @param AcceptanceTester $I Actor object.
+	 *
+	 * @return void
+	 */
+	public function _before( AcceptanceTester $I ) {
+		$this->rollback = $I->bootstrapWith( [ __CLASS__, '_enableAuthorship' ] );
```

Comment:
Idea for a later addition, maybe we could add this to the helper where `bootstrapWith` is defined, so you could do a class level bootstrap as well as individual test level.

Reply:
Not sure I'm following, can you elaborate a bit more ? Tests currently does not extend any class, so not sure which class you're referring to.